docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+  app:
```

Comment:
Consider adding an image name here so we can test that the artifact was generated correctly.